### PR TITLE
DM-43378: Check repo path when using run_rc2_subset script

### PR DESCRIPTION
**Pull request by leeskelvin**

### CODE REVIEW

File: bin/run_rc2_subset.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ RC2_SUBSET_PIPE=${RC2_SUBSET_PIPE:-$DRP_PIPE_DIR/pipelines/HSC/DRP-RC2_subset.ya
 echo "[RC2_SUBSET_PIPE] Pipeline def YAML: $RC2_SUBSET_PIPE"
 
 # Check that required file paths exist.
-paths=($RC2_SUBSET_PIPE)
+paths=($RC2_SUBSET_REPO $RC2_SUBSET_PIPE)
```
Comment (erykoff) [inline bin/run_rc2_subset.sh:43]: This check does not work for either of these.  If they are empty then it checks if the root directory exists, so always succeeds.

**Review by erykoff — CHANGES_REQUESTED** [review]

File: bin/run_rc2_subset.sh (hunk 1 above)
Comment (leeskelvin) [reply bin/run_rc2_subset.sh:43]: I posted an example failure on Jira. Namely, just running this script with only `lsst_distrib` set up and nothing else returns:
```
ERROR: Path does not exist: /SMALL_HSC
```
Are you unable to reproduce that?

Comment (leeskelvin) [reply bin/run_rc2_subset.sh:43]: (Note: it _will_ succeed if there exists a directory named `SMALL_HSC` in the root directory, but the chances of that are probably not worth considering here)

Comment (erykoff) [reply bin/run_rc2_subset.sh:43]: You are correct; this is my mistake because the PR diff doesn't show the full context.

**Review by erykoff — APPROVED** [review]

**Pull request merged**